connector.go

```diff
@@ -55,16 +55,13 @@ func NewConnector(options ...Option) (*Connector, error) {
 		return nil, fmt.Errorf("could not load RPC list from config file")
 	}
 
-	// Create a proto registry client to dynamically load protobuf definitions.
-	prc := protoregistry.NewClient(conf.GetString("protoregistry.host"))
-
 	c := &Connector{
-		manifest:         LoadManifest(),
-		env:              kafkautils.Env(conf.GetString("kafka.env")),
-		kafkaUrl:         conf.GetString("kafka.url"),
-		Health:           healthcheck.NewHandler(),
-		RPCMap:           rpcMap,
-		ProtoRegistryCli: prc,
+		manifest:          LoadManifest(),
+		env:               kafkautils.Env(conf.GetString("kafka.env")),
+		kafkaUrl:          conf.GetString("kafka.url"),
+		Health:            healthcheck.NewHandler(),
+		RPCMap:            rpcMap,
+		protoRegistryHost: conf.GetString("protoregistry.host"),
```

Comment:
Why is the change?

Reply:
This seems to add extra level of complexity when it is only a URL that is needed. Is there another reason that I am missing for why you chose to add an object here?
JS library needs to be the same as this one so I wanted to simplify it.